dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -144,21 +154,18 @@ class TaskGridFilter extends FilterPropertySource {
     if (!_taskProperty.matches(qualifiedTask.task!)) {
       return false;
     }
-    const Map<String, String> showOSs = {
-      'showMac': 'mac',
-      'showWindows': 'windows',
-      'showiOS': 'ios',
-      'showLinux': 'linux',
-      'showAndroid': 'android',
-    };
-    for (MapEntry<String, String> os in showOSs.entries) {
-      if (!_allProperties[os.key]?.value && qualifiedTask.task!.toLowerCase().contains(os.value)) {
-        return false;
-      }
-    }
 
-    // Unrecognized stages always pass.
-    return true;
+    LinkedHashMap<String, bool> orderedOSFilter = LinkedHashMap<String, bool>.of({
+      'ios': _allProperties['showiOS']?.value ?? false,
+      'android': _allProperties['showAndroid']?.value ?? false,
+      'mac': _allProperties['showMac']?.value ?? false,
+      'windows': _allProperties['showWindows']?.value ?? false,
+      'linux': _allProperties['showLinux']?.value ?? false,
+    });
+    return orderedOSFilter.entries
+            .firstWhereOrNull((MapEntry<String, bool> os) => qualifiedTask.task!.toLowerCase().contains(os.key))
+            ?.value ??
+        true;
```

Comment:
Let's move the existing comment.
```suggestion
        true; // Unrecognized stages always pass.
```

Reply:
Will do that in a minute, additionally I want to update the Readme.md to fix a typo. Is that fine?